source/programming/introduction.rst

```diff
@@ -1,2 +1,28 @@
 前言
 ====
+
+日常科研工作中，我们需要使用计算机语言编写程序和脚本，以自动完成相应的数据处理或科学计算。
+因此，需要了解\ **常见的计算机语言**\  、\ **编译和链接**\ 以及\ **版本控制**\ 等。
+
+计算机语言一般可以分为两大类。第一类是需要编译和链接的\ **编译语言**\ ，如 C 语言、
+Fortran 语言；第二类是不需编译便可直接执行的\ **解释型语言**\ ，也称\ **脚本语言**\ ，如
+Python 语言、Julia 语言。通常来说，使用编译语言编写的代码运行速度更快，
+但是编写代码的时间成本更多，且需要我们了解更多的编程基础；脚本语言易于编写，
+开源的软件库和模块更多，但是运行速度较慢。
+
+我们推荐至少掌握一门编译语言，同时至少掌握一门脚本语言。使用哪种编译语言并不重要，熟悉掌握一门
+编译语言，其他编译语言也就可以很快地学会和上手了。至于脚本语言，我们推荐使用目前至少是在地震学领域
+最流行的 Python 语言，其官方模块、第三方模块和软件包众多，基本可以满足我们日常的科研工作需求。
```

Comment:
We should rethink it. I have the feeling that people now can do researches using Python only, even though they know nothing about C and Fortran. So perhaps we should say "a scripting language is required, but a compiling language is also recommended".

Reply:
okay